CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - #1285, Abort on wrong database password - @qu4tro
 - #790, Allow override of OpenAPI spec through `root-spec` config option - @steve-chavez
 - #1308, Accept `text/plain` and `text/html` for raw output - @steve-chavez
+- #1339, Add websearch_to_tsquery support - @herulume
```

Comment:
Move it to the `Unreleased` section`above pls.

https://github.com/PostgREST/postgrest/blob/ea970554491557d79408ee2193d8a5f08d2984ea/CHANGELOG.md#added

Reply:
Also change the issue from #1339 to #1243.

Reply:
Sorry for the confusion.